basis/http/http.factor

```diff
@@ -152,6 +159,7 @@ redirects ;
         <url>
             H{ } clone >>query
         >>url
+        <url> >>proxy-url
```

Comment:
I think we should default ``proxy-url`` to ``f`` (mostly since it's never used, and also because I think setting an "invalid" proxy should be a ``invalid-proxy`` error when you try and make a request)?